extensions/linux/dictionary.json

```diff
@@ -17,6 +17,11 @@
       "caption": "Effective Group ID",
       "description": "The effective group under which this process is running.",
       "type": "integer_t"
+    },
+    "kernel_ver": {
+        "caption": "Kernel Version",
+        "description": "The kernel release version of the operating system. Determined from the <code>uname -r</code> command output on Unix-based systems.",
```

Comment:
This is debatable but we may not be able to give strong implementation guidance and simply say "For Example" similar to elsewhere in the OCSF schema. The reason is some input data sources will be things like logs where it is not easy to control the format of information (i.e., cannot run `uname -r` like CSE agent can).